pkg/chrootarchive/archive.go

```diff
@@ -48,16 +46,20 @@ func untarHandler(tarArchive io.Reader, dest string, options *archive.TarOptions
 		}
 	}
 
-	r := ioutil.NopCloser(tarArchive)
+	r := tarArchive
```

Comment:
@c0b let's split this change to separate PR, so we can just merge your `CmdStream` fix.